companion/manifest.json

```diff
@@ -0,0 +1,31 @@
+{
+	"id": "phillips-hue",
```

Comment:
Keep this type in order to overwrite the old module

Reply:
you can change this if you add the old spelling to the `legacyIds` field further down